shoplist/src/main/java/com/netguru/codereview/network/ShopListApiMock.kt

```diff
@@ -0,0 +1,42 @@
+package com.netguru.codereview.network
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+import com.netguru.codereview.network.model.ShopListResponse
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+class ShopListApiMock : ShopListApi {
+    override suspend fun getShopLists(): List<ShopListResponse> =
+        coroutineScope {
+            List(9999) { index ->
+                ShopListResponse(
+                    list_id = index.toString(),
+                    userId = index,
+                    listName = "ListName$index"
+                )
+            }
+        }
+
+    override suspend fun getShopListItems(listId: String): List<ShopListItemResponse> =
+        coroutineScope {
+            Thread.sleep(2)
```

Comment:
```suggestion
            delay(2000)
```

You have to use delay instead of Thread.slepp